commit 385e5fef2c00cc978352f50b46a67bd138743d6a
Author: Emmanuel Pescosta <emmanuelpescosta099@gmail.com>
Date:   Tue Dec 3 19:00:28 2013 +0100

    Fix Bug 328262 - rename bug if you cancel when folder already exists
    
    Only connect the renamingFailed signal if there is no item with the new name in the model yet.
    
    BUG: 328262
    FIXED-IN: 4.11.5
    REVIEW: 114228

diff --git a/dolphin/src/views/dolphinview.cpp b/dolphin/src/views/dolphinview.cpp
index 20bc9f5..db9d8d2 100644
--- a/dolphin/src/views/dolphinview.cpp
+++ b/dolphin/src/views/dolphinview.cpp
@@ -1492,7 +1492,9 @@ void DolphinView::slotRoleEditingFinished(int index, const QByteArray& role, con
             }
 
             KonqOperations* op = KonqOperations::renameV2(this, oldUrl, newName);
-            if (op) {
+            if (op && !newNameExistsAlready) {
+                // Only connect the renamingFailed signal if there is no item with the new name
+                // in the model yet, see bug 328262.
                 connect(op, SIGNAL(renamingFailed(KUrl,KUrl)), SLOT(slotRenamingFailed(KUrl,KUrl)));
             }
         }
